pkg/querier/http.go

```diff
@@ -193,6 +199,49 @@ func (q *QueryHandlers) Render(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func (q *QueryHandlers) Settings(w http.ResponseWriter, r *http.Request) {
```

Comment:
We don't need this. We can just register the new connect-go services.

Reply:
👍